Dockerfile

```diff
@@ -74,83 +75,75 @@ ARG ADD_DEB_PACKAGES="\
+    python3-shapely \
     python3-tinydb \
     python3-xarray \
     python3-zarr \
     python3-mapscript \
-    "
+    python3-pytest \
+    python3-pyld"
 
 # ENV settings
 ENV TZ=${TZ} \
     LANG=${LANG} \
     DEBIAN_FRONTEND="noninteractive" \
     DEB_BUILD_DEPS="\
-      gcc-aarch64-linux-gnu \
-      curl \
-      python3-dev \
-      software-properties-common \
-      unzip" \
+    curl \
+    unzip" \
     DEB_PACKAGES="\
-      gunicorn \
-      libsqlite3-mod-spatialite \
-      locales \
-      locales-all \
-      python3-dateutil \
-      python3-distutils \
-      python3-flask \
-      python3-flask-cors \
-      python3-gevent \
-      python3-greenlet \
-      python3-pip \
-      python3-setuptools \
-      python3-shapely \
-      python3-tz \
-      python3-unicodecsv \
-      python3-wheel \
-      python3-yaml \
-      ${ADD_DEB_PACKAGES}"
+    locales \
+    tzdata \
+    gunicorn \
+    python3-dateutil \
+    python3-flask \
+    python3-flask-cors \
+    python3-gevent \
+    python3-greenlet \
+    python3-pip \
+    python3-tz \
+    python3-unicodecsv \
+    python3-yaml \
+    ${ADD_DEB_PACKAGES}"
+
+WORKDIR /pygeoapi
+ADD . /pygeoapi
 
 # Install operating system dependencies
 RUN \
     apt-get update -y \
     && apt-get upgrade -y \
-    && apt-get install -y --fix-missing --no-install-recommends ${DEB_BUILD_DEPS}  \
-    && add-apt-repository ppa:ubuntugis/ubuntugis-unstable \
-    && apt-get --no-install-recommends install -y ${DEB_PACKAGES} \
-    && update-locale LANG=${LANG} \
-    && echo "For ${TZ} date=$(date)" && echo "Locale=$(locale)"
-
-# OGC schemas local setup
-WORKDIR /schemas.opengis.net
-RUN \
-    curl -O http://schemas.opengis.net/SCHEMAS_OPENGIS_NET.zip \
-    && unzip ./SCHEMAS_OPENGIS_NET.zip "ogcapi/*" -d /schemas.opengis.net \
-    && rm -f ./SCHEMAS_OPENGIS_NET.zip
+    && apt-get --no-install-recommends install -y ${DEB_PACKAGES} ${DEB_BUILD_DEPS}  \
+    && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 \
+    && echo "For ${TZ} date=$(date)" && echo "Locale=$(locale)"  \
 
-WORKDIR /pygeoapi
-RUN mkdir -p /pygeoapi/pygeoapi
-# Add files required for pip/setuptools
-ADD requirements*.txt setup.py README.md /pygeoapi/
-ADD pygeoapi/__init__.py /pygeoapi/pygeoapi/
+    # temporary remove
+    # && add-apt-repository ppa:ubuntugis/ubuntugis-unstable \
+
+    # OGC schemas local setup
+    && mkdir /schemas.opengis.net \
+    && curl -O http://schemas.opengis.net/SCHEMAS_OPENGIS_NET.zip \
+    && unzip ./SCHEMAS_OPENGIS_NET.zip "ogcapi/*" -d /schemas.opengis.net \
+    && rm -f ./SCHEMAS_OPENGIS_NET.zip \
 
-RUN \
     # Install pygeoapi
     # Optionally add development/test/doc packages
-    if [ "$BUILD_DEV_IMAGE" = "true" ] ; then pip3 install -r requirements-dev.txt; fi \
-    # Install pygeoapi providers
-    && pip3 install -r requirements-provider.txt \
+    && if [ "$BUILD_DEV_IMAGE" = "true" ] ; then pip3 install -r requirements-dev.txt; fi \
```

Comment:
This can be taken out: we don't need the DEV Image anymore (as we have pytest+pyld now).